source/includes/_under-the-hood.html.md

```diff
@@ -1,8 +1,8 @@
-## Under the Hood
+# Under the Hood
```

Comment:
The last time we talked about this section it was going to be:

A.I Engine Guide ->
1. About AI
2. Under the Hood
3. Versioning (but we didn't know at the time that this was just 2 sentences)

Right now we just have Under the Hood content so I am OK with splitting it up.